ewkcoffea/modules/corrections.py

```diff
@@ -155,17 +155,13 @@ def run3_muons_sf_attach(muons,year,id_method,iso_method):
     # Evaluate the ID SF
     ceval = correctionlib.CorrectionSet.from_file(fname)
     sf_id_flat_nom = ceval[id_method].evaluate(abseta_flat,pt_flat,"nominal")
-    sf_id_flat_syst = ceval[id_method].evaluate(abseta_flat,pt_flat,"syst")
-    # Finding the syst uncertaintities
-    sf_id_flat_hi = sf_id_flat_nom + sf_id_flat_syst
-    sf_id_flat_lo = sf_id_flat_nom - sf_id_flat_syst
+    sf_id_flat_hi  = ceval[id_method].evaluate(abseta_flat,pt_flat,"systup")
```

Comment:
So just to confirm I'm understanding properly, these up and down variation already include the nominal piece (i.e. you don't have to do the "nominal + up" and "nominal - down" type of a thing)?

Reply:
Yes, the POG now provides the actual up and down systematic